src/Client/Log.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace HelloFresh\Stats\Client;
+
+
+use HelloFresh\Stats\Client;
+use HelloFresh\Stats\Incrementer;
+use HelloFresh\Stats\State;
+use HelloFresh\Stats\Timer;
+use Psr\Log\LoggerInterface;
+
+class Log extends Base implements Client
```

Comment:
I think this naming maybe a little off this looks more like a factory class. Then again I have no clue what this is for :smile: